src/main/java/teammates/ui/controller/ControllerServlet.java

```diff
@@ -39,6 +41,11 @@
     private static final Logger log = Logger.getLogger();
 
     @Override
+    public void init() throws ServletException {
+        TimeHelper.setSystemTimeZoneIfRequired();
```

Comment:
This servlet will be loaded before all the other servlets. The call affects the entire application (even the timestamp of the console log messages changes).